Auth0/Authentication.swift

```diff
@@ -1133,18 +1167,55 @@ public extension Authentication {
     ```
     Auth0
        .authentication(clientId: clientId, domain: "samples.auth0.com")
-       .tokenExchange(withAppleAuthorizationCode: authCode, scope: "openid profile offline_access", audience: "https://myapi.com/api, fullName: credentials.fullName)
+       .tokenExchange(withAppleAuthorizationCode: authCode,
+           scope: "openid profile email",
+           audience: "https://myapi.com/api",
+           fullName: credentials.fullName)
        .start { print($0) }
     ```
 
     - parameter authCode: Authorization Code retrieved from Apple Authorization
-    - parameter scope: requested scope value when authenticating the user. By default is 'openid profile offline_access'
+    - parameter scope: Requested scope value when authenticating the user. By default is `openid profile offline_access`
     - parameter audience: API Identifier that the client is requesting access to
     - parameter fullName: The full name property returned with the Apple ID Credentials
 
     - returns: a request that will yield Auth0 user's credentials
     */
-    func tokenExchange(withAppleAuthorizationCode authCode: String, scope: String? = nil, audience: String? = nil, fullName: PersonNameComponents? = nil) -> Request<Credentials, AuthenticationError> {
+    func tokenExchange(withAppleAuthorizationCode authCode: String, scope: String? = "openid profile offline_access", audience: String? = nil, fullName: PersonNameComponents? = nil) -> Request<Credentials, AuthenticationError> {
```

Comment:
Should we move this to a constant?

Reply:
Currently none of the default param values in the Authentication API client have been extracted to constants. However, for the next major I plan on reviewing and consolidating those default values, and maybe extracting them. But not right now.